packages/flutter/lib/src/material/slider_theme.dart

```diff
@@ -536,6 +564,9 @@ class PaddleSliderValueIndicatorShape extends SliderComponentShape {
 
   // Radius of the top lobe of the value indicator.
   static const double _topLobeRadius = 16.0;
+  // Baseline size of the label text. This is the size that the value indicator
```

Comment:
probably best to steer away from the term "baseline" when referring to text, since it has an existing precise meaning

Reply:
Ooh, good point.